app/_layout.tsx

```diff
@@ -53,7 +53,7 @@ function RootLayoutNav() {
         <Stack>
           <Stack.Screen name="(tabs)" options={{ headerShown: false }} />
           <Stack.Screen name="modal" options={{ presentation: 'modal' }} />
-          <Stack.Screen name="upload/image/camera" options={{ headerShown: false }} />
+          <Stack.Screen name="upload/text/textbox" options={{ headerShown: false }} />
```

Comment:
I think we mentioned this before but i think it'd be best to not add the code directly to Layout and the index page before. However, I think itd be better for us to keep this in mind for the future so we don't have to deal with so many merge conflicts.